dune_api_scripts/queries/raw_app_data.sql

```diff
@@ -0,0 +1,3 @@
+CREATE OR REPLACE VIEW
+    dune_user_generated.cow_protocol_raw_app_data (hash, content)
```

Comment:
Why don't we attach an Enviroment variable here as well? Why do you only do it for: cow_protocol_app_data?

Reply:
We can also do this. It probably better this way (I did mention this in the PR description). Will do now.